### DISCUSSION TITLE
Exposure Time Finder -> different settings for layers with same z positioning

### DISCUSSION THREAD

**Discussion by xFluor** [sn4k3/UVtools#820] [Q&A] [answered]
Hello! 

I am using the exposure time finder to print several objects (pattern loaded model option) with different exposure time.
And I saw that the total print time could be highly different if I check the box "use different settings for layers with same z positioning", in fact much faster if cheched.

How does the option work ? because I thought I would be the contrary. 

And as I am unsing the new Elegoo Saturn 3 Ultra which has troubles with crushing prints of all flat calibration samples, I want to be sure that there is no more fantasy on the z axis, if the option is checked or not checked. 

Thanks for helping !

Best regards

**Comment by sn4k3** [comment, marked as answer]
That option enables the change of some parameters (in same positioned layers), which one is lift height. By almost eliminate the lifts on same layers you are saving time which can save big amount of time if your lift height is big or slow. But it also allows you to set what value you want, and in that case, you can input a large lift which will have the opposite effect. My recommendation is to leave that ON with the defaults. It will never hurt and if not able it will print as same.

**Reply by xFluor** [reply]
OK all clear now ! 
With this, I will let it ON for sure as my height is slow.
Thanks ;)

**Reply by bryanseger** [reply]
xFluor,

I'm curious to hear what your impressions are on the Saturn 3 and UVtools. Every new Mars 4 variant has some type of issue with UVtools (and we believe it's the firmware on the printer, not UVtools itself). I suspect the new line of Saturn 3 machines have the same issue, but we don't have the physical machines to say one way or the other, and I haven't seen anyone else comment on it.

Going off memory, as an example, I think I recall that the first 'strip' of an exposure test followed whatever bottom layer exposure time is set instead of whatever exposure time was specified by the UVtools. Also, I recall something going awry when we had different values for the lift heights/speeds 1&2 and retract heights/speeds 1&2 (using TSMC) to where the build head would continuously move the correct amount for 'lift' but would omit part of 'retract', giving grossly incorrect Z height placement.


Anyways, just wanted to hear your thoughts on the Saturn 3 and UVtools and if you're noticing anything abnormal.
